platform/ios/README.md

```diff
@@ -6,10 +6,15 @@ The polyPod app for iOS devices
 
 ### Prerequisites
 
-Before building for Android, you need to build the shared components and
+Before building for iOS, you need to build the shared components and
 Features - simply follow the build instructions in
 [../../README.md](../../README.md).
 
+As a summary, you will need to:
+
+* Build the shared components in [`core`](../../core). You will probably be interested only in the `iOS` part
+* The polyPod core and features are written in JavaScript. You will need to build and bundle them by running `./build.js` from the root directory of this repository. Please follow instructions [there](../../README.md) to install all needed prerrequisites. 
+
```

Comment:
That seems redundant with the paragraph above - it refers people specifically to the build instructions in ../../README.md. Then you do it _again_ down here, including some details like that one needs to run `build.js`. Also, there's no need to manually build _core_, the top-level README describes how to get the iOS and/or Android cores built automatically.